### PR TITLE
[Merged by Bors] - feat: Simple `Finpartition` lemmas

### PR DESCRIPTION
**Pull request by YaelDillies**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Partition/Equipartition.lean [hunk 1]
```diff
@@ -40,6 +40,11 @@ theorem isEquipartition_iff_card_parts_eq_average :
 
 variable {P}
 
+@[simp]
```
Comment (b-mehta) [inline Mathlib/Order/Partition/Equipartition.lean:43]: I'm not sure that this should be `simp` given that `P.IsEquipartition` is already simp-normal?

**Review by b-mehta — APPROVED** [review]
bors d+

File: Mathlib/Order/Partition/Equipartition.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Order/Partition/Equipartition.lean:43]: It's often the case that `A` is simp nf, but `¬ A` is not, right? At any rate, I don't really mind so let's remove it for now.
```suggestion
```

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**